### DISCUSSION TITLE
Export Package not work

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#12529] [Q&A]
When export theme and other content types from server 

![image](https://user-images.githubusercontent.com/5202291/193263513-7dfeae80-7689-4afc-8514-648528f3fd08.png)

to sync to local because local is behind from server, after download the Recipe.json and replace the server url with local url and then import package on local:

![image](https://user-images.githubusercontent.com/5202291/193264876-fd796bfb-6972-4cf1-bbff-1535aa2a1d12.png)

got error see below

![image](https://user-images.githubusercontent.com/5202291/193264766-b2318916-c47c-4635-91ce-0d6dee8c8875.png)

Please advise

**Comment by ns8482e** [comment]
May be you need update content export deployment step when you export

**Reply by saleemhaq** [reply]
> May be you need update content export deployment step when you export

you mean to say I create another deployment type and add these things again and then export and import on local then? @ns8482e

**Reply by ns8482e** [reply]
I mean, while looking at your screenshot “All Content” that adds, I guess instead of that you need Update
